build.sbt

```diff
@@ -115,20 +115,20 @@ lazy val bcsBackend = (project in backendRoot / "bcs")
   .dependsOn(ossFileSystem % "test->test")
   .dependsOn(services % "test->test")
 
+lazy val CwlV1Project = LocalProject("cwlV1_0LanguageFactory")
 lazy val engine = project
   .withLibrarySettings("cromwell-engine", engineDependencies, engineSettings)
   .dependsOn(backend)
   .dependsOn(ossFileSystem)
   .dependsOn(gcsFileSystem)
-  .dependsOn(wdl)
-  .dependsOn(cwl)
   .dependsOn(core % "test->test")
   .dependsOn(backend % "test->test")
   // In the future we may have a dedicated test backend like the `TestLocalAsyncJobExecutionActor`.
   // For now, all the engine tests run on the "Local" backend, an implementation of an impl.sfs.config backend.
   .dependsOn(sfsBackend % "test->compile")
   .dependsOn(gcsFileSystem % "test->test")
   .dependsOn(ossFileSystem % "test->test")
+  .aggregate(CwlV1Project)
```

Comment:
What does this do? I've only ever seen [aggregate on the root sub-project](http://www.scala-sbt.org/1.0/docs/Multi-Project.html#Dependencies). You can also include that link as it documents the difference between `dependsOn` and `aggregate`.

Reply:
This was an artifact me messing around trying to trick it to import the CWL code for tests only. I'm going to remove this.